services/core-web/src/components/common/DocumentTable.js

```diff
@@ -96,7 +151,23 @@ export const DocumentTable = (props) => {
         </div>
       ),
     },
-  ];
+    !IN_PROD() && {
```

Comment:
@taraepp I got a bit carried away and introduced a `featureFlag` utility in the common package to centralize the flags and have them defined in one place 🏕️ 🔥 

Regardless of whether we end up defining them in ENV variables or using a Feature Flag tool, I think this will make that transition easier. Let me know your thoughts on this and I'll follow up with another PR to move the TSF flags over to this as well

Reply:
🙌 
Oh I definitely meant just a comment but I do like this! Looks flexible enough to change the function in the featureFlags file to use a library or ENV variable down the road. 👍 

Heads-up: there is also a Projects feature flag still in effect. Before when I found that I just put a comment with FEATURE FLAG: PROJECTS on those instances.

Reply:
Started out as just a comment, buuuut couldn't help myself 😅